Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
    ~~~~~~~~~^
        nb,
        ^^^
    ...<4 lines>...
        **kwargs,
        ^^^^^^^^^
    )
    ^
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/asyncio/base_events.py", line 725, in run_until_complete
    return future.result()
           ~~~~~~~~~~~~~^^
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
        cell, index, execution_count=self.code_cells_executed + 1
    )
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# create a table and assign it to the object table_1
table_1 <- sleep_40 %>%
  # select the variables to include
  select(gender,
         age,
         phys_active,
         smoke_now,
         alcohol_day_num,
         regular_marij,
         bmi_group,
         hypertension,
         sleep_hrs_night) %>%
           display_html(tbl_summary(missing_text = "(Missing)",
                # define clear labels for each variable
                label = list(gender ~ "Gender",
                age ~ "Age",
                phys_active ~ "Regularly Physically Activity",
                smoke_now ~ "Current Smoker",
                alcohol_day_num ~ "Alcohol Drinks per Day",
                regular_marij ~ "Regular Marijuana User",
                bmi_group ~ "BMI",
                hypertension ~ "Hypertension",
                sleep_hrs_night ~ "Sleep (hrs/night)")))

# display the table
table_1
------------------


Error in display_html(., tbl_summary(missing_text = "(Missing)", label = list(gender ~ : could not find function "display_html"
Traceback:

1. .handleSimpleError(function (cnd) 
 . {
 .     watcher$capture_plot_and_output()
 .     cnd <- sanitize_call(cnd)
 .     watcher$push(cnd)
 .     switch(on_error, continue = invokeRestart("eval_continue"), 
 .         stop = invokeRestart("eval_stop"), error = NULL)
 . }, "could not find function \"display_html\"", base::quote(display_html(., 
 .     tbl_summary(missing_text = "(Missing)", label = list(gender ~ 
 .         "Gender", age ~ "Age", phys_active ~ "Regularly Physically Activity", 
 .         smoke_now ~ "Current Smoker", alcohol_day_num ~ "Alcohol Drinks per Day", 
 .         regular_marij ~ "Regular Marijuana User", bmi_group ~ 
 .             "BMI", hypertension ~ "Hypertension", sleep_hrs_night ~ 
 .             "Sleep (hrs/night)")))))

